lib/interface_quda.cpp

```diff
@@ -3618,10 +3623,14 @@ void invertMultiShiftQuda(void **_hp_x, void *_hp_b, QudaInvertParam *param)
 
   // grow resident solutions to be big enough
   for (int i=solutionResident.size(); i < param->num_offset; i++) {
+    printfQuda("Adding vector %d to solutionsResident\n", i);
```

Comment:
Unguarded `printfQuda` without any verbosity check.  I guess this line should either be deleted or guarded by a `QUDA_DEBUG_VERBOSE` check.

Reply:
da1904c5e